packages/host/tests/acceptance/code-submode/inspector-test.ts

```diff
@@ -299,6 +299,59 @@ const importsSource = `
   }
 `;
 
+const reExportSource = `
+  import {
+    CardDef,
+    FieldDef,
+    BaseDef as BDef,
+    contains,
+  } from 'https://cardstack.com/base/card-api';
+  import StringCard from 'https://cardstack.com/base/string';
+  import NumberCard from 'https://cardstack.com/base/number';
+
+  export const exportedVar = 'exported var';
+
+  export { StringCard as StrCard };
+
+  export { FieldDef as FDef, CardDef, contains, BDef };
+
+  export * from './in-this-file'; //Will not display inside "in-this-file"
+
+  export default NumberCard;
```

Comment:
there is another form of reexport that you dont have here:
```
export { foo } from "foo";
```
this will be similar to the `export * ` example you have where the reexport never enters the module scope--and should not be displayed in the schema editor

Reply:
Resolved. I think the second syntax is wrong. The correct way to re-export default from another module is 

```
export { default as Foo } from 'foo';
```

Our schema analysis handles both cases. I added test for these

```
export { Foo as Bar  } from 'foo';
export { default as Foo } from 'foo-exporting-default';
```